user/pages/04.Reference/08.network/docs.en.md

```diff
@@ -157,3 +157,17 @@ The DNS zone specfied in `--dns-domain` must be created and owned by the same pr
 The record name will be generated from the VM name. Note that some symbols like underscore (`_`) or spaces ( ) are not allowed in host names, and will be removed from the host name. When a Floating IP address is detached from a VM the corresponding A and PTR records are deleted automatically.
 
 When using this approach, Floating IPs themselves don't need to be created using the `--dns-domain` and `--dns-name` options.
+
+## IPv6
+
+You have the possibility to allocate IPv6 Global Unicast Addresses for your virtual subnets, where supported. IPv6 is supported in the Dual-Stack mode, which means that single virtual network may have both IPv4 and IPv6 subnets. Ports of virtual machines connected to such network, will get both IPv4 and IPv6 addresses. IPv6-only networks are currently not fully supported, due to limitations in the Metadata service. While you may stil run such setups, your virtual machines will not be able to retrieve public SSH keys or use other features that Metadata service provides.
+
+There is no possibility to use Floating IPs in IPv6 address family, which means that you cannot allocate a Floating IP from IPv6 address pool, neither you can have a Floating IP pointing at an IPv6 address.
+
+During the creation of IPv6 subnet, you may enter a CIDR manually, or allocate a globally routable /64 prefix from a public address pool. Subnets with manually set CIDR can not send traffic or be reached to/from the Internet. Inter-VM communication is not restricted in any way. If you choose to allocate IPv6 prefix from an address pool instead, you will get a globally routable /64 prefix reachable from the Internet for your load balancers and VMs. You can not choose the exact IPv6 subnet you will get, but as long as your subnet is not deleted, allocated range can not be changed. The range is assigned to you and does not interfere with any customers, so it is safe to assume that full /64 range is yours, when you configure VPN or Firewalls.
+
+! Please note that every virtual machine or load balancer with an IPv6 address in globally routable subnet is immediately reachable from the Internet, so plan your security groups accordingly.
+
+### IPv6 addressing
+
+When IPv6 range is allocated and configured on a subnet, you should set `IPv6 RA mode` and `IPv6 address mode`. These two control how IPv6 address is allocated for the port that is connected to IPv6-enabled subnet. We recommend setting both of these options to `dhcpv6-stateless`, which will generate a safe random IPv6 address, that will be acquired by a virtual machine via DHCP. Altough other combinations of these options are available, we don't recommend setting them, unless you know what you are doing and how ICMPv6 RA works. Since these options affect cloud-init, DHCP client, and ICMPv6 ND protocol configuration inside your virtual machines, changing of them should always go together with reconfiguration of operating system inside virtual machine.
```

Comment:
```suggestion
When an IPv6 range is allocated and configured on a subnet, you should set `IPv6 RA mode` and `IPv6 address mode`. These two control how the IPv6 address is allocated for the port that is connected to the IPv6-enabled subnet. We recommend setting both of these options to `dhcpv6-stateless`, which will generate a safe random IPv6 address, that will be acquired by a virtual machine via DHCP. Altough other combinations of these options are available, we don't recommend setting them, unless you know what you are doing and how ICMPv6 RA works. Since these options affect cloud-init, DHCP client, and ICMPv6 ND protocol configuration inside your virtual machines, changing them should always go together with reconfiguration of the operating system inside the virtual machine.
```